back/engines/commercial/analysis/spec/lib/auto_tagging_task_spec.rb

```diff
@@ -17,10 +17,155 @@
         state: 'succeeded',
         progress: nil
       })
-      controversial_tag = Analysis::Tag.find_by(tag_type: AutoTaggingMethod::Controversial::TAG_TYPE)
+      controversial_tag = Analysis::Tag.find_by(tag_type: Analysis::AutoTaggingMethod::Controversial::TAG_TYPE)
       expect(controversial_tag).to be_present
       expect(idea1.tags).to include(controversial_tag)
       expect(idea2.tags).not_to include(controversial_tag)
     end
   end
+
+  describe 'PlatformTopic auto_tagging' do
+    it 'works' do
+      analysis = create(:analysis)
+      att = create(:auto_tagging_task, analysis: analysis, state: 'queued', auto_tagging_method: 'platform_topic')
```

Comment:
(Nice to have: we could maybe simplify the creation of tagging tasks by adding a few traits to the factory.)